apps/frontend/src/renderer/components/onboarding/OllamaModelSelector.tsx

```
@@ -439,136 +381,128 @@ export function OllamaModelSelector({
-         })}
-       </div>
+      <div className="space-y-2">
+        {models.map(model => {
+          const isSelected = selectedModel === model.name;
+          const isCurrentlyDownloading = isDownloading === model.name;
+          const progress = downloadProgress[model.name];
+
+          return (
+            <div
+              key={model.name}
+              role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)